src/Symfony/Component/Debug/ErrorHandler.php

```diff
@@ -109,6 +109,21 @@ class ErrorHandler
     private $displayErrors = 0x1FFF;
 
     /**
+     * wether we should also listen to FATAL_ERROR levels (hhvm only).
+     *
+     * @var bool
+     */
+    private static $fatalErrors = false;
```

Comment:
Sorry, I wasn't clear enough: I meant having an interview here: the resolved bitfield of all fatal error levels